ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/RedHat.yml

```diff
@@ -47,6 +45,8 @@ Build_Tool_Packages:
   - zip
 
 Additional_Build_Tools_NOT_RHEL8:
+  - libdwarf-devel                # now in CodeReady Linux Builder (CRB) repo
```

Comment:
I'm worried about expanding this section. While I accept that this the name wasn't defined in this PR, `NOT_RHEL8` doesn't feel right for this. It was ok when it was just NTP (Since that has an explicit replacement in RHEL8) but these don't. I'd rather it was called somthign that implied `<8` or `<=7` as the same will likely apply to 8 and later.

Reply:
Well it's with RHEL8 that the packages have been moved, but I get your point that it doesn't take into account future releases..... However, the RedHat statement about packages ending up in CRB is that they're unsupported, so does that mean they might disappear altogether in a future release?

I think the naming is fairly clear currently, and there's also other sections like `Java_NOT_RHEL6_PPC64` (with the comment "Not RHEL8 either") which have a similar problem, so if we change one, we should review the rest also.  Would you be willing to let this go to a fresh issue where new names can be discussed and agreed?

Reply:
Also, I'd like to name the Centos change in this PR too for Will, and that would have similar naming issues.

Reply:
`Java_NOT_RHEL6_PPC64` isn't so bad IMHO because we won't supported anything earlier than 6, so it's clear

Reply:
OK, I've raised #1757 to do this properly once we've agreed what names should be used.